bikes/models.py

```diff
@@ -1,38 +1,217 @@
 from django.utils import timezone
 from django.urls import reverse #Used to generate URLs by reversing the URL patterns
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.translation import ugettext as _
+from localflavor.us.models import USStateField
+from localflavor.us.models import USZipCodeField
+
+
+class People(models.Model):
+    ROLE_CHOICES = (
+        (4, 'Admin'),
+        (3, 'Manager'),
+        (2, 'Volunteer'),
+        (1, 'User'),
+        (0, 'No Waiver'))
+    FOUND_CHOICES = (
+        (0, 'Internet'),
+        (1, 'FaceBook'),
+        (2, 'Friend'),
+        (3, 'Event'),
+        (4, 'Flyer'),
+        (5, 'Random'),
+        (6, 'Other'))
+    WHY_CHOICES = (
+        (0, 'Work on own bike'),
+        (1, 'Work on friends bike'),
+        (2, 'Buy bike'),
+        (3, 'Get parts'),
+        (4, 'Volunteer'),
+        (5, 'Other'))
+    GENDER_CHOICES = (
+        (1, 'Female'),
+        (2, 'Male'),
+        (3, 'TransMale'),
+        (4, 'TransFemale'),
+        (5, 'Gender NonConforming'),
+        (6, 'Something Else'),
+        (7, 'Decline to Answer'))
+    PRONOUN_CHOICES = (
+        (1, 'He'),
+        (2, 'She'),
+        (3, 'They'),
+        (4, 'Ze'),
+        (5, 'N/A'),
+        (6, 'No Preference'))
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    add_date = models.DateField(auto_now_add=True, primary_key=True)
+    first = models.CharField(max_length=30, blank=True)
+    last = models.CharField(max_length=30, blank=True)
+    email = models.EmailField()
+    phone = models.CharField(max_length=20, blank=True, default='')
+
+    address_1 = models.CharField(_("address"), max_length=128)
+    address_2 = models.CharField(_("address cont'd"), max_length=128, blank=True)
+    city = models.CharField(_("city"), max_length=64, default="Corvallis")
+    state = USStateField(_("state"), default="OR")
+    zip_code = USZipCodeField(_("zip code"), default="97330")
+
+    whyhere_id = models.IntegerField(choices=WHY_CHOICES,
+                                      default=None, null=True)
+    foundhow_id = models.IntegerField(choices=FOUND_CHOICES,
+                                       default=None, null=True)
+    volunteer_hrs = models.IntegerField(default=0)
+    bench_hrs = models.IntegerField(default=0)
+    birth_date = models.DateField(null=True, blank=True)
+    gender_id = models.IntegerField(choices=GENDER_CHOICES,
+                                      default=None, null=True)
+    pronoun_id = models.IntegerField(choices=PRONOUN_CHOICES,
+                                      default=None, null=True)
+    role_id = models.PositiveSmallIntegerField(choices=ROLE_CHOICES,
+                                               null=True, blank=True)
+
+    def __str__(self):  # __unicode__ for Python 2
+        return self.user.username
+
+@receiver(post_save, sender=User)
```

Comment:
these two listeners cause problems when running:
`python manage.py createsuperuser`

Reply:
We shouldn't really be using `python manage.py createsuperuser` should we?